mne_bids/tests/test_write.py

```diff
@@ -752,6 +762,7 @@ def test_edf(_bids_validate):
     # Also cover iEEG
     # We use the same data and pretend that eeg channels are ecog
     ieeg_raw = raw.copy()
+    ieeg_raw.set_montage(None)
```

Comment:
Needed to add to get rid of the 'captrak' head montage from EEG.

Reply:
Could you add this as a comment to the code? :)